packages/override-tools/src/scripts/generateManifest.ts

```diff
@@ -19,6 +19,7 @@ import {diff_match_patch} from 'diff-match-patch';
 import {getInstalledRNVersion} from '../ReactVersion';
 
 const WIN_PLATFORM_EXT = /\.win32|\.windows|\.windesktop/;
+const WS_CHARS = /\s/g;
```

Comment:
>WS_CHARS [](start = 6, length = 8)

(nit) can you rename it to "WHITESPACE"? "SPACE_CHARS"? Or something similar?

Reply:
But then I would need to do more ugly line breaking to hit eslints 80 char line limit 😛

Reply:
I can change this though.